src/App.scss

```diff
@@ -52,11 +50,15 @@
   position: fixed !important;
 }
 
-@media screen and (min-width: 1255px) {
+@media screen and (min-width: 768px) {
   .map-container {
     .mapboxgl-map {
       margin-top: 120px;
-      min-height: calc(110dvh - 311px);
+      min-height: calc(100dvh - 230px);
     }
   }
 }
+
+.footer-logo-map {
+  width:60% !important;
```

Comment:
Unless something else is setting the width, we shouldn't be using `!important` on this style. In general, we should avoid using `!important` unless absolutely necessary.